### PR TITLE
privoxy: update to release 3.0.34 + PCRE2

### PR DESCRIPTION
**Pull request by Ansuel**
Update to release 3.0.34 to permit easier backport of PCRE2 support.

Backport required patch for PCRE2 support and move to PCRE2 library as
PCRE is now EOL and won't receive any security updates anymore.

### CODE REVIEW

File: net/privoxy/Makefile [hunk 1]
```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=privoxy
-PKG_VERSION:=3.0.33
-PKG_RELEASE:=4
+PKG_VERSION:=3.0.34
+PKG_RELEASE:=1
 
 PKG_SOURCE:=privoxy-$(PKG_VERSION)-stable-src.tar.gz
```
Comment (BKPepe) [inline net/privoxy/Makefile:14]:
```suggestion
PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-stable-src.tar.gz
```

File: net/privoxy/Makefile [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
 PKG_SOURCE_URL:=@SF/ijbswa
-PKG_HASH:=04b104e70dac61561b9dd110684b250fafc8c13dbe437a60fae18ddd9a881fae
+PKG_HASH:=e6ccbca1656f4e616b4657f8514e33a70f6697e9d7294356577839322a3c5d2c
 PKG_BUILD_DIR:=$(BUILD_DIR)/privoxy-$(PKG_VERSION)-stable
```
Comment (BKPepe) [inline net/privoxy/Makefile:17]:
```suggestion
PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)-stable
```

**Review by BKPepe — COMMENTED** [review]
I do think, we need one more patch:
https://www.privoxy.org/gitweb/?p=privoxy.git;a=commit;h=e73b93ea9ad1f3e980bd78ed3ebf65dedbb598a2

And maybe.... what about to disable PCRE for privoxy? Does it make sense? Is it required/handy to use?

**Comment by Ansuel** [comment]
@BKPepe i think we already have that patch as 103... for disabling pcre no idea

**Comment by BKPepe** [comment]
Oh... 🤦 🤦 You are right! Sorry. Okay. LGTM

**Review by BKPepe — APPROVED** [review]

**Comment by Ansuel** [comment]
@BKPepe made the requested changes. think we can merge!

**Pull request merged**